azure-storage/spring-cloud-starter-stream-sink-wasb/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-cloud-starter-stream-sink-wasb</artifactId>
+    <name>spring-cloud-starter-stream-sink-wasb</name>
+    <description>Spring Cloud Stream Windows Azure Storage Blob (WASB) Sink</description>
```

Comment:
Nit-pick, but Microsoft calls it like "Blob Storage": https://azure.microsoft.com/en-us/documentation/articles/storage-java-how-to-use-blob-storage/